VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -154,11 +154,11 @@ private void setCurrentUnit(CourseComponent component){
 
         environment.getDatabase().updateAccess(null, selectedUnit.getId(), true);
 
-        CourseComponent parent = component.getParent();
+        CourseComponent parentSectionComponent = component.getParent().getParent();
```

Comment:
Does this need a null check? They're not expected right now, but we might see course trees with different structures in the future. Better to do a little future proofing if it's easy.